docs/user/dql/basics.rst

```diff
@@ -155,6 +155,46 @@ Result set:
 |  Nanette|   Bates|
 +---------+--------+
 
+One can also provide meta-field name(s) in ``SELECT`` clause to retrieve reserved-fields (beginning with underscore) from OpenSearch documents.
```

Comment:
Maybe mention that these fields are not included in `SELECT *`, but could be used for search (does it work though?).

Reply:
Yes. `SELECT *` won't return _id, but `SELECT *, _id` will